src/index.ts

```diff
@@ -19,6 +19,16 @@ circumstances are right.
 import type { ClientBase as PgClientBase } from 'pg';
 import { type SocketDefaults } from './shims/net';
 
+// Ensure we are very explicit while using these apis. This ensures more type safety
+// specially since this library is made to run both in the browser and node.js environments.
+declare global {
```

Comment:
this was mainly to avoid adding dom types globally to the tsconfig. It makes it more type safe